src/scss/_brand.scss

```diff
@@ -47,13 +47,22 @@
 			nav-text: oColorsGetPaletteColor('black'),
 			nav-background: oColorsGetPaletteColor('slate-white-5'),
 			nav-border: oColorsGetPaletteColor('black-20'),
-			nav-hover-background: oColorsGetPaletteColor('slate-white-15')
+			nav-hover-background: oColorsGetPaletteColor('slate-white-15'),
 		),
 		'supports-variants': ()
 	));
 } @elseif oBrandGetCurrentBrand() == 'whitelabel' {
 	@include oBrandDefine('o-header-services', 'whitelabel', (
-		'variables': (),
+		'variables': (
+			logo: null,
+			top-text: oColorsGetPaletteColor('white'),
+			top-background: oColorsGetPaletteColor('black'),
+			nav-text: oColorsGetPaletteColor('black'),
+			nav-background: oColorsGetPaletteColor('white'),
+			nav-border: oColorsGetPaletteColor('black'),
+			nav-hover-background: oColorsGetPaletteColor('white'),
+			underline-hover: true
```

Comment:
The boolean is acting more like a distinct variant that should be configured under `supports-variants` than a fallback. What if the conditional was based on the existence of `nav-hover-background` instead? If `nav-hover-background` is not configured then it would be sensible to show underlines. 

Something like this:
```scss
@mixin _oHeaderServicesHover() {
	color: _oHeaderServicesGet('nav-text');
	position: relative;

	@if not oBrandGet('nav-hover-background') {
		// underlines
	} @else {
		// background
	}
}
```